src/styles/commonStyles.tsx

```diff
@@ -125,21 +125,36 @@ export const commonStyles = makeStyles()((theme) => {
     submitButton: {
       textTransform: 'lowercase',
       backgroundColor: primaryBtnBg,
-      color: '#FFFFFF',
+      color: theme.palette.common.white,
       height: 56,
       fontSize: 18,
       fontWeight: 600,
       borderRadius: 10,
       marginTop: 45,
       ':disabled': {
-        backgroundColor: '#FFFFFF',
+        backgroundColor: theme.palette.common.white,
         border: '2px solid #FB8F67',
         color: primaryBtnBg,
       },
       '&: hover': {
         backgroundColor: primaryBtnBg,
       },
     },
+
+    noticeButton: {
+      textTransform: 'lowercase',
+      backgroundColor: theme.palette.primary.light,
+      color: theme.palette.common.white,
+      height: 60,
+      width: 180,
+      fontSize: 18,
+      fontWeight: 600,
+      borderRadius: 10,
+      '&: hover': {
+        backgroundColor: theme.palette.primary.main,
+      },
+    },
+
```

Comment:
It looks like this `noticeButton` repeated the styles from `submitButton`

Reply:
Maybe it would be better. I think commonStyles needs refactoring. But it should be along with changes to the corresponding components. In my opinion, it should be a separate task